src/components/AlertStatus/AlertStatus.tsx

```diff
@@ -3,7 +3,8 @@ import { GrafanaTheme2 } from '@grafana/data';
 import { Icon, IconButton, LinkButton, Tooltip, useStyles2, useTheme2 } from '@grafana/ui';
 import { css, cx } from '@emotion/css';
 
-import { AlertSensitivity, Check, PrometheusAlertsGroup, ROUTES } from 'types';
+import { AlertSensitivity, Check, PrometheusAlertsGroup } from 'types';
```

Comment:
I like that you moved the routing types away from the huge `types.ts` file. Is the idea to do this for most components in upcoming PRs?

Reply:
Yes :D